### PR TITLE
fix(scripts):deploy meta-service failed because of no pv definition

### PR DESCRIPTION
**Pull request by hantmac**
Signed-off-by: hantmac <hantmac@outlook.com>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

- fix(scripts):deploy meta-service failed because of no pv definition

## Changelog

- Bug Fix: deploy meta-service failed because of no pv definition

### CODE REVIEW

File: scripts/kubernetes/meta-standalone.yaml [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+  labels:
+    type: local
+spec:
+  storageClassName: local-storage
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/data"
```
Comment (Xuanwo) [inline scripts/kubernetes/meta-standalone.yaml:15]: Is it safe to share the same host path with other pods?

Comment (hantmac) [reply scripts/kubernetes/meta-standalone.yaml:15]: > Is it safe to share the same host path with other pods?

I'm afraid not safe, change it to another maybe better.

Comment (Xuanwo) [reply scripts/kubernetes/meta-standalone.yaml]: I have noticed that other services have similar problems. Are you willing to change them at the same time?

For naming, I prefer:

- meta: `/mnt/databend/meta`
- minio: `/mnt/databend/minio`
- component_name: `/mnt/databend/component_name`

Does this look good to you?

Comment (hantmac) [reply scripts/kubernetes/meta-standalone.yaml]: > I have noticed that other services have similar problems. Are you willing to change them at the same time?
> 
> For naming, I prefer:
> 
> * meta: `/mnt/databend/meta`
> * minio: `/mnt/databend/minio`
> * component_name: `/mnt/databend/component_name`
> 
> Does this look good to you?

Yeah,mostly LGMT. BTW, where is the `component_name ` service? Or what is the `component_name ` mean?

**Pull request merged**